dev/com.ibm.ws.jdbc/resources/com/ibm/ws/rsadapter/resources/IBMDataStoreAdapterNLS.nlsprops

```diff
@@ -271,26 +271,22 @@ DSRA4011.tran.none.iso.switch.unsupported=DSRA4011E: The JDBC driver does not su
 DSRA4011.tran.none.iso.switch.unsupported.explanation=The JDBC specification does not support switching to an isolation level of TRANSACTION_NONE.
 DSRA4011.tran.none.iso.switch.unsupported.useraction=Avoid switching to an isolation level of TRANSACTION_NONE.
 
-DSRA4012.ignored.datasource.config.used=DSRA4012I: When using Oracle UCP the following data source properties are ignored: statementCacheSize and validationTimeout. Use the equivalent Oracle UCP functionality.
-DSRA4012.ignored.datasource.config.used.explanation=The statementCacheSize and validationTimeout data source properties are not valid when using Oracle UCP.
-DSRA4012.ignored.datasource.config.used.useraction=Remove the data source properties and instead use the equivalent Oracle UCP functionality.
+DSRA4012.ignored.datasource.config.used=DSRA4012I: When using Oracle UCP the following data source properties are ignored: {0}.
+DSRA4012.ignored.datasource.config.used.explanation=The listed data source properties are not valid when using Oracle UCP.
+DSRA4012.ignored.datasource.config.used.useraction=Remove the data source properties and instead use the equivalent Oracle UCP functionality, if available.
 
-DSRA4013.ignored.connection.manager.config.used=DSRA4013I: When using Oracle UCP the following connection manager properties are ignored: agedTimeout, connectionTimeout, maxIdleTime, maxPoolSize, minPoolSize, purgePolicy, reapTime, maxConnectionsPerThread, maxConnectionsPerThreadLocal. Use the equivalent Oracle UCP functionality.
-DSRA4013.ignored.connection.manager.config.used.explanation=The listed connection manager properties are not valid when using Oracle UCP.
-DSRA4013.ignored.connection.manager.config.used.useraction=Remove the connection manager properties and instead use the equivalent Oracle UCP functionality.
+DSRA4013.ignored.connection.manager.config.used=DSRA4013I: When using Oracle UCP the following connection manager properties are valid: {0}.
+DSRA4013.ignored.connection.manager.config.used.explanation=When using Oracle UCP, only the listed connection manager properties are valid.
+DSRA4013.ignored.connection.manager.config.used.useraction=Remove the invalid connection manager properties and instead use the equivalent Oracle UCP functionality, if available.
```

Comment:
is there any information we can give about action to take if no UCP functionality is available?

Reply:
There isn't really any action they can take in the case that they're trying to use function available in Liberty's connection pool but not Oracle's.

Reply:
Ok, looks good then.